packages/main/src/SegmentedButton.ts

```diff
@@ -84,6 +89,26 @@ class SegmentedButton extends UI5Element {
 	@property({ defaultValue: undefined })
 	accessibleName?: string;
 
+	/**
+	 * Defines the component selection mode.
+	 *
+	 * <br><br>
+	 * <b>The available values are:</b>
+	 *
+	 * <ul>
+	 * <li><code>Single</code></li>
+	 * <li><code>Multi</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.main.types.SegmentedButtonMode}
+	 * @defaultvalue "Single"
+	 * @public
+	 * @name sap.ui.webc.main.SegmentedButton.prototype.mode
+	 * @since 1.14.0
+	 */
+	@property({ type: SegmentedButtonMode, defaultValue: SegmentedButtonMode.Single })
+	mode!: SegmentedButtonMode;
```

Comment:
There is one novelty that is not communicated, that's why you could not know.
Here is full background: https://github.com/SAP/ui5-webcomponents/pull/6856

In short,
For public APIs - properties and params of public methods, we use template literals when enum is expected. For example:

mode!: `${ListMode}`;  or in your case mode!: `${SegmentedButtonMode}`;

This is done to allow usage of the enum values in Typescript application without importing the enum itself. For example:
sb.mode = "Single" would not be possible, it should have been sb.mode = "SegmentedButtonMode.Single" and the enum imported and used

With the template literal syntax - the app can use like without the Enum and directly sb.mode = "Single".

Reply:
Done